client/www/js/services.js

```diff
@@ -93,16 +93,29 @@
     });
   };
 
+  var updateTeam = function(teamIndex, gameCode) {
+    var teamUpdate = { teamIndex: teamIndex};
```

Comment:
Super nit-picky, but can you fix the spacing in the object teamUpdate to {teamIndex: teamIndex}?